test/components/invariant_test.dart

```diff
@@ -18,12 +21,23 @@ main() {
   test("invariant -- getCoords", () {
     var feature2 = Feature<LineString>(geometry: line1);
     expect(() => getCoords(null), throwsA(isA<Exception>()));
-    expect(() => getCoords(feature1), throwsA(isA<Exception>())); 
+    expect(
+        getCoords([
+          Position.of([119.32, -8.7]),
+          Position.of([119.55, -8.69]),
+          Position.of([119.51, -8.54]),
+          Position.of([119.32, -8.7])
+        ]),
+        equals([
+          Position.of([119.32, -8.7]),
+          Position.of([119.55, -8.69]),
+          Position.of([119.51, -8.54]),
+          Position.of([119.32, -8.7])
+        ]));
+    expect(() => getCoords(feature1), throwsA(isA<Exception>()));
+    expect(() => getCoords(feature1), throwsA(isA<Exception>()));
     var coords = getCoords(feature2);
```

Comment:
we checked for:
- simple types such as `null` to throw
- list of `Position`s to return
- features with
  - simple geometry types like `Point` to throw
  - geometry types with a list of coordinates like `LineString` to return
- What we should do to have covered everything: create a test case, with a LineString or Polygon, **not wrapped** with a `Feature`